src/config/dynamic-theme-fixes.config

```
@@ -1739,6 +1739,19 @@ CSS
 
 ================================
 
+bitwarden.com
+
+INVERT
+img[alt="Github Logo"]
+body > div > section:nth-child(4) > div > div:nth-child(1) > div > blockquote > footer > cite > img
```

What about to simplify it?
```suggestion
img[src$="forbes.png"]
```